### ISSUE TITLE
Images should be deletable

### ISSUE THREAD

**Issue by mwanner2** [localwiki/localwiki#738]
It appears that once an image has been added to a page, it cannot be deleted from the list of addable images that appears when one clicks the Image button.

This is not good: an image could be a copyright violation, or entirely inappropriate.

**Comment by philipn** [comment]
Yes, I definitely agree. Tracking this at https://github.com/localwiki/localwiki-backend-server/issues/66 for now, and marking it as urgent.
